docs/mecsimcalc-library.md

```diff
@@ -653,3 +653,112 @@ Displaying Image
 Downloading Image
 {{ outputs.download }}
 ```
+
+## Quiz Toolkit
+### append_to_google_sheet
+
+[**[Source]**](https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.6/mecsimcalc/quiz_utils.py#L12) 
+
+```python
+append_to_google_sheet(
+    service_account_info = {...},
+    spreadsheet_id = "123abc...",
+    values = [["name", 12837, ...]],
+)
```

Comment:
Could you also show the optional values here. All the other examples show the optional arguments with their default value